tests/models/test_models.py

```diff
@@ -366,6 +366,84 @@ def test_partial_patch_trial_metadata(clean_db):
     TrialMetadata._patch_trial_metadata(TRIAL_ID, metadata_patch)
 
 
+@db_test
+def test_trial_metadata_get_summaries(clean_db, monkeypatch):
+    """Check that trial data summaries are computed as expected"""
+    monkeypatch.setattr(
+        "cidc_api.models.models.DISABLE_METADATA_VALIDATION_ON_INSERT", True
+    )
+
+    # Add some trials
+    records = [{"fake": "record"}]
+    tm1 = {
+        **METADATA,
+        "protocol_identifier": "tm1",
```

Comment:
maybe add a comment saying that this is purposefully overriding the `protocol_identifier` in `METADATA`